lib/rmt/cli/clean.rb

```diff
@@ -121,18 +121,15 @@ def dangling_packages_list(repo_base_dir, repomd_file)
     packages = (actual_packages - expected_packages).sort
 
     packages.map do |file|
-      next nil unless File.exist?(file)
-
-      # Only remove files if they were not recently created
-      file_stat = File.stat(file)
-      next nil if (Time.current - file_stat.mtime) < DANGLING_FILE_MINIMUM_AGE
+      file_status = File.stat(file)
```

Comment:
Won't `File.stat('/path/doesnt/exist')` fail here, which is unhandled?  And it would render the `File.exist?` call at line 160 inside `eligible_for_cleaning?` moot - File.exist is just a wrapper around stat() anyway.

Reply:
You actually make an excellent point. I changed the behavior on 020f16f.